openmdao/core/total_jac.py

```diff
@@ -1242,8 +1242,18 @@ def compute_totals(self):
                             varlist = '(' + ', '.join([name for name in par_deriv[key]]) + ')'
                             print('Solving color:', key, varlist)
                         else:
-                            print('Solving variable: {0} \nIn mode: {1} \n'
-                                  'Sub Indices: {2}'.format(key, mode, inds))
+                            if key == '@simul_coloring':
+                                local_inds = imeta['coloring']._local_indices(
+                                    inds=inds, mode=self.mode)
+                                print('In mode: {0}, Solving variable(s):'.format(
```

Comment:
This first print statement is now identical in both branches of the if block, so it could be pulled outside of the if block.